elements/search/nuxeo-results-view.js

```diff
@@ -315,7 +315,19 @@ Polymer({
   _getFilterCount() {
     if (this._params) {
       // subtract the number of original parameters (this._paramsCount)
-      return Object.keys(this._params).length - this._paramsCount - ('highlight' in this._params ? 1 : 0);
+      return (
+        Object.keys(this._params).filter((p) => {
+          if (this._params[p]) {
```

Comment:
Shouldn't we be checking for
```suggestion
          if (this._params[p] != null) {
```
Instead? Empty strings are valid params, right?

Reply:
I don't think so, when doing so I end up with a badge showing `1` active param once I have manually cleared an input and it is not consistent with the result set.